### ISSUE TITLE
Template's style blocks should overload base template super

### ISSUE THREAD

**Issue by eviltnan** [barseghyanartur/django-admin-timeline#7]
In the case, when the base_site.html template is overloaded and uses extra styles, timeline won't see it, because `extrastyle` and `extrahead` blocks overload the parent's ones without calling `{{ block.super}}`

Should become something like:

```
{% block extrastyle %}
    {{ block.super }}
    <link rel="stylesheet" type="text/css" href="{% static 'admin_timeline/css/admin_timeline.css' %}"/>
    <link rel="stylesheet" type="text/css"
          href="{% static 'admin_timeline/libs/multiple-select/multiple-select.css' %}"/>
{% endblock extrastyle %}
{% block extrahead %}
    {{ block.super }}
    <script type="text/javascript" src="{% url 'admin:jsi18n' %}"></script>
    <script src="{% static 'admin/js/core.js' %}"></script>
    <script src="{% static 'admin_timeline/libs/jquery-1.11.2.min.js' %}"></script>
    <script src="{% static 'admin_timeline/libs/multiple-select/jquery.multiple.select.js' %}"></script>
    <script src="{% static 'admin_timeline/js/admin_timeline.js' %}"></script>
{% endblock extrahead %}
{% block coltype %}colMS{% endblock %}
```

**Comment by barseghyanartur** [comment]
@eviltnan:

Thanks for bringing this up. I'll check it soon.

**Comment by barseghyanartur** [comment]
@eviltnan:

Fixed in version 1.6.1.
